packages/core/src/actions/invoke.ts

```diff
@@ -67,7 +67,7 @@ export function invoke<
             type,
             params: {
               ...invokeDef,
-              ref: interpret(behavior, { id })
+              ref: interpret(behavior, { id, parent: actorContext?.self })
```

Comment:
q: why this is needed now and why it wasn't needed before? I think this is the correct change - but I wonder what changed that prompted it now

Reply:
Before, this was directly added to `actorRef.parent = this` or something like that.